edudoc/delegate-handbook/delegate-handbook.md

```diff
@@ -1298,15 +1298,15 @@ The following are steps to create a business card using VistaPrint. Screenshots
 4. Click "Make more edits" on the bottom
 5. Using the font Catamaran and the reference images as a guide, position four text boxes with the following text:
 
-   World Cube Association (Size 8, Bolded)
+::::: {.indent} World Cube Association (Size 8, Bolded)
```

Comment:
::::: {.indent} should start on a separate line and World Cube... should be on the next line

Reply:
Fixed in latest commit.